docs/changes/newsfragments/5888.improved_driver

```diff
@@ -0,0 +1,3 @@
+Improved Stahl BS Voltage Source driver:
+- accepts more floating-point formats in response from instrument
+- allows connection to serial instrument over TCP/IP connection
```

Comment:
This file is missing a blank line at the end. I tried fixing it but it seems like you have not allowed maintainers to push to your branch. If you install the precommit hooks these kind of small things would be fixed automatically

Reply:
added newline